tsconfig.json

```diff
@@ -4,29 +4,29 @@
 
     /* Basic Options */
     // "incremental": true,                   /* Enable incremental compilation */
-    "target": "es5",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */
```

Comment:
Is het bewust dat alle tabjes weg zijn?  Of gewoon ge-autoformat?

Reply:
Auto format goes brrrrr